File: test_cases/c/claude/12.c
#include <assert.h>
#include <stdbool.h>
#include <stddef.h>

bool isScramble(char* s1, char* s2);

int main(void) {
    assert(isScramble("great", "rgeat") == true);
    assert(isScramble("abcde", "caebd") == false);
    assert(isScramble("a", "a") == true);
    assert(isScramble("abc", "abc") == true);
    assert(isScramble("abcdefghijklmnopq", "efghijklmnopqcadb") == false);
    return 0;
}

File: test_cases/python/claude/12.py
def run_tests(solution_module):
    f = solution_module.isScramble
    # identical strings
    assert f('abc', 'abc') is True
    # length 1, same char
    assert f('a', 'a') is True
    # length 1, different char
    assert f('a', 'b') is False
    # known scramble
    assert f('great', 'rgeat') is True
    # known non-scramble
    assert f('abcde', 'caebd') is False
